lib/logstash/outputs/amazon_es.rb

```diff
@@ -288,16 +288,16 @@ def receive(event)
       @retry_queue_not_full.wait(@retry_flush_mutex) while @retry_queue.size > @retry_max_items
     }
 
-    event['@metadata']['retry_count'] = 0
+    event.set('[@metadata][retry_count]', 0)
 
     # Set the 'type' value for the index.
     type = if @document_type
-             event.sprintf(@document_type)
-           elsif @index_type # deprecated
-             event.sprintf(@index_type)
-           else
-             event["type"] || "logs"
-           end
+      event.sprintf(@document_type)
```

Comment:
The indentation change here makes it look like the diff is bigger than it is (and means elsif/else aren't aligned with if).